### PR TITLE
Force DoVi on browser.xboxOne as edgeUWP says it can't play it

### PR DESCRIPTION
**Pull request by RaafatAkkad**
Unsure if DoVi can be used with profile 8 on Xbox but video play

https://ott.dolby.com/codec_test/index.html
Screenshot 2024-06-10 at 8 50 16 PM

### CODE REVIEW

**Comment by RaafatAkkad** [comment]
> ## [Quality Gate Failed](https://sonarcloud.io/dashboard?id=jellyfin_jellyfin-web&pullRequest=5669) **Quality Gate failed**
> Failed conditions  [C Reliability Rating on New Code](https://sonarcloud.io/dashboard?id=jellyfin_jellyfin-web&pullRequest=5669) (required ≥ A)
> 
> [See analysis details on SonarCloud](https://sonarcloud.io/dashboard?id=jellyfin_jellyfin-web&pullRequest=5669)
> 
>  Catch issues before they fail your Quality Gate with our IDE extension  [SonarLint](https://www.sonarsource.com/products/sonarlint/features/connected-mode/?referrer=pull-request)

Unrelated to my change

File: src/scripts/browserDeviceProfile.js [hunk 1]
```diff
@@ -244,12 +244,14 @@ function supportedDolbyVisionProfilesHevc(videoTestElement) {
     if (videoTestElement.canPlayType) {
         if (videoTestElement
             .canPlayType('video/mp4; codecs="dvh1.05.09"')
-            .replace(/no/, '')) {
+            .replace(/no/, '')
+            || browser.xboxOne) {
```
Comment (thornbill) [inline src/scripts/browserDeviceProfile.js:248]: I would move this to the beginning of the function and just do

```js
if (browser.xboxOne) return [ 5, 8 ];
```

Comment (RaafatAkkad) [reply src/scripts/browserDeviceProfile.js]: Thanks done.

**Review by thornbill — APPROVED** [review]

**Comment by RaafatAkkad** [comment]
Thanks @thornbill
Until `options.supportsDolbyVision` returns true (when https://github.com/jellyfin/jellyfin-uwp/pull/46 is merged), that code path will not be executed.

Given that jellyfin 10.9 has been out for a while, I'm going to retarget https://github.com/jellyfin/jellyfin-web/pull/5670 to master and this can be enabled to the next major release.

**Pull request merged**